app/src/components/menu.ts

```diff
@@ -94,6 +98,14 @@ export function generateMenu(
         accelerator: 'CmdOrCtrl+C',
         role: 'copy',
       },
+      {
+        label: 'Copy as Plain Text',
+        accelerator: 'CmdOrCtrl+Shift+C',
+        click: (): void => {
+          const text = clipboard.readText('selection');
+          clipboard.writeText(getPlainTextWithoutFormatting(text), 'clipboard');
```

Comment:
Let's document that the bulk of the work happens here
```suggestion
          // We use clipboard.readText to strip down formatting
          const plainText = clipboard.readText('selection');
          clipboard.writeText(getPlainTextWithoutFormatting(plainText), 'clipboard');
```